staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1145,6 +1155,8 @@ message UpdateContainerResourcesRequest {
     // container resources updating. This can be used for specifying experimental
     // resources to update or other options to use when updating the container.
     map<string, string> annotations = 4;
+    // Resource configuration for the container.
+    ContainerResources resources = 5;
```

Comment:
?  the fields in container resources are already in this.. don't see a good reason to pass them twice?

Reply:
Yes, I also think this field is redundant. As UpdateContainerResourcesRequest define, you can use this two field to replace ContainerResources resources = 5;

```
// Resource configuration specific to Linux containers.
LinuxContainerResources linux = 2;
// Resource configuration specific to Windows containers.
WindowsContainerResources windows = 3;
```

Reply:
hmm.. this was in the KEP from before WindowsContainerResources got added. 
Based on the code [here](https://github.com/kubernetes/kubernetes/pull/102884/files#diff-1a7a9a00739661ddc6e3dd02d6552b6e83240436ec7bc69e1e9b6bce6eb44caeR623), I think I can chuck it.. I'll take a very close look after the OSS conference. (Last week of june)

Reply:
@mikebrow @halfrost Please review CL https://github.com/kubernetes/kubernetes/commit/66f6239802aad21e2fa25cec558fa2a11bc8bc5a which removes this spurious field that was in the design which predates 1.20 when WindowsContainerResources was added.

Once I have your confirmation about this change and sign-off on CRI, I'll create another PR to fix the KEP (remove [UpdateContainerResourcesRequest changes](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#cri-changes) described there).

Reply:
that works, thanks!

Reply:
great!

Reply:
fixed and merged in a separate CRI changes commit. resolving conversation.